plutus-ledger-api/test/Spec/Versions.hs

```diff
@@ -76,7 +76,7 @@ testBuiltinVersions = testGroup "builtins"
          assertBool "not in l2,Vasil" $ isRight $ V2.deserialiseScript vasilPV serialiseDataExScript
          assertBool "not in l3,future" $ isRight $ V3.deserialiseScript futurePV serialiseDataExScript
     , testCase "bls,keccak256,blake2b224 only available in l3,Future and after" $
-         for_ [blsExScript, keccak256ExScript, blake2b224ExScript] $ \script -> do
+         for_ [blsExScript, keccak256ExScript, blake2b224ExScript, ripemd160ExScript] $ \script -> do
```

Comment:
I'm not totally sure if `ripemd160` should be in here, but these tests need to be reworked anyway.

Reply:
I can drop it if you wish :-)